fuel-tx/src/transaction.rs

```diff
@@ -954,6 +1013,16 @@ pub mod field {
         fn bytecode_root_offset_static() -> usize;
     }
 
+    pub trait BlobId {
+        fn blob_id(&self) -> &fuel_types::BlobId;
+        fn blob_id_mut(&mut self) -> &mut fuel_types::BlobId;
```

Comment:
Because every other trait here has it as well, is why I added it. Even the other traits seem to use `_mut` methods only for testing, maybe we should `#[cfg(tests)]` those as well?

Reply:
Yeah. That makes sense to me. Either that or call YAGNI.

Reply:
Actually, that's a breaking change to modify those traits. I'm going to keep this as-is for now, but we can open a follow-up issue if you want to.